### ISSUE TITLE
start_docker.sh redownloads models every time I launch

### ISSUE THREAD

**Issue by foodornt** [w-okada/voice-changer#753]
| Question                  | Answer           | Question                 | Answer                      |
|--------------------------|------------------|--------------------------|-----------------------------|
| Issue Type               | Feature Request | vc client version number | commit 5b43daa              |
| OS                       | Arch linux      | GPU                      | 3050ti                      |
| Clear setting            | no              | Sample model             | no                          |
| Input chunk num          | no              | Wait for a while         | The GUI successfully launched. |
| read tutorial            | yes             | Extract files to a new folder. | yes                    |
| Voice Changer type       | RVC             | Model type               | I guess pytorch?           |

### Situation

Basically running start_docker.sh is gonna download base models (the anime girl thingies) on every launch

### logs on terminal
![image](https://github.com/w-okada/voice-changer/assets/40924367/680be020-d381-4917-a45f-f8049f2bf94e)

**Comment by w-okada** [comment]
It's definitely a hassle, isn't it? 
Maybe it would be good to set it up to mount the local folder. 
I'll give it a try.

**Comment by w-okada** [comment]
Done.
Try the latest version v.1.5.3.13. And read tutorial.

**Comment by foodornt** [comment]
Superior! Works like butter now ❤️
